Nodejs/Tests/AnalysisTests/AnalysisFile.cs

```diff
@@ -131,6 +131,20 @@ static class Analysis {
             List<AnalysisFile> files = new List<AnalysisFile>();
             foreach (var file in Directory.GetFiles(directory, "*", SearchOption.AllDirectories)) {
```

Comment:
From what I understand about this tool, it is purpose is kind of developer tool which run analysis on the common npm modules and verifies how analysis goes on then. This is not actual automated testing, but rather tool for capturing feedback. I maybe wrong about that, but I use it that way, trying figure out what's wrong. Again @DinoV maybe give clearer explanation.